src/vivarium/framework/results/manager.py

```diff
@@ -48,16 +50,23 @@ def __init__(self) -> None:
 
     @property
     def name(self) -> str:
-        """The name of this ResultsManager."""
         return self._name
 
     def get_results(self) -> Dict[str, pd.DataFrame]:
         """Return the measure-specific formatted results in a dictionary.
 
-        NOTE: self._results_context.observations is a list where each item is a dictionary
-        of the form {lifecycle_phase: {(pop_filter, stratifications): List[Observation]}}.
+        Notes
+        -----
+        self._results_context.observations is a list where each item is a dictionary
+        of the form {lifecycle_phase: {(pop_filter, stratification_names): List[Observation]}}.
         We use a triple-nested for loop to iterate over only the list of Observations
-        (i.e. we do not need the lifecycle_phase, pop_filter, or stratifications).
+        (i.e. we do not need the lifecycle_phase, pop_filter, or stratification_names
+        for this method).
+
+        Returns
+        -------
+        Dict[str, pd.DataFrame]
```

Comment:
I guess we have to mention the type here? It's a bit annoying to have to do so, since it's redundant with the typing of the return type of the function itself.

Reply:
I'm completely happy breaking the "rules" and removing types. Maybe I'll do that in the separate PR that makes sure everything is numpy style